daemon/cmd/loadbalancer.go

```diff
@@ -109,6 +109,7 @@ func (h *putServiceID) Handle(params PutServiceIDParams) middleware.Responder {
 			fmt.Errorf("the service provided is already registered with ID %d, please use that ID instead of %d",
 				id, frontend.ID))
 	} else if err != nil {
+		log.WithError(err).Error("Error while inserting service in LB map")
```

Comment:
Why is this needed? The caller should already log the error, no? 
You can easily test this using `cilium service update ...` command (first create a local-redirect service, and then try to create the same service with clusterIP type). Ping me on Slack if you need any help.

Reply:
@aditighag I probably misinterpreted your review above:
> Thanks for the PR. You are missing the error check for CLI - [http://github.com/cilium/cilium/blob/master/daemon/cmd/loadbalancer.go#L106](https://github.com/cilium/cilium/blob/master/daemon/cmd/loadbalancer.go#L106). This is the only place where we should log it as an error, the rest should be converted to debug level.

I reproduced the error as you suggested:
```
# cilium service update --local-redirect --backends 1.2.3.4:8080 --frontend 5.6.7.8:8080 --id 42
Creating new service with id '42'
Added service with 1 backends
# cilium service update --backends 1.2.3.4:8080 --frontend 5.6.7.8:8080 --id 42
Updating existing service with id '42'
Error: Cannot add/update service: [PUT /service/{id}][500] putServiceIdFailure  local-redirect service exists for frontend {{5.6.7.8 {TCP 8080} 0} 42}, skip update for svc / &{BackendAddresses:[0xc000844fc0] Flags:0xc000847f80 FrontendAddress:0xc000844c40 ID:42 UpdateServices:false}
```

It's printing the error message to `stderr` and it exits with an error code:
https://github.com/cilium/cilium/blob/master/cilium/cmd/service_update.go#L223

Is there anything else we should do here?